r55/src/exec.rs

```diff
@@ -38,7 +38,12 @@ pub fn deploy_contract(db: &mut InMemoryDB, bytecode: Bytes) -> Address {
     }
 }
 
-pub fn run_tx(db: &mut InMemoryDB, addr: &Address, calldata: Vec<u8>) {
+pub struct TxResult {
+    pub output: Vec<u8>,
+    pub logs: Vec<Log>  
+}
+
+pub fn run_tx(db: &mut InMemoryDB, addr: &Address, calldata: Vec<u8>) -> TxResult {
```

Comment:
Not for this PR but we should refactor the handler in a simple top-level export that you can just import, and we should move all of the stuff in a separate `bin` so that the library code is separated